src/Runner.js

```diff
@@ -172,16 +161,63 @@ function run(transformFile, paths, options) {
         reject(e.message);
       });
     });
-  } else if (!fs.existsSync(transformFile)) {
-    process.stderr.write(
-      colors.white.bgRed('ERROR') + ' Transform file ' + transformFile + ' does not exist \n'
-    );
-    return;
-  } else {
-    return transform(transformFile);
+  } else if (fs.existsSync(transform)) {
+    let stats = fs.lstatSync(transform);
+
+    if (stats && stats.isFile(transform)) {
+      // handle file
+      return transform;
+    } else if (stats && stats.isDirectory(transform)) {
+      let directory = path.join(transform, 'transforms');
+      let file = path.join(transform, 'transformjs');
+
+      // handle directory
+      if (fs.existsSync(directory) && fs.lstatSync(directory).isDirectory()) {
+        return fs.readdirSync(directory);
+      } else if (fs.existsSync(file) && fs.lstatSync(file).isFile()) {
+        return file;
+      } else {
+        return fs.readdirSync(transform);
+      }
+    } else {
+      // handle other fs objects
+      return transform;
+    }
+  }
+
+  try {
+    // handle npm package as directory
+    return getTransform(require.resolve(transform));
```

Comment:
I don't think that works as planned. Or at least it works differently than you mentioned in the README. `require.resolve(...)` will either resolve to a file path or it will throw an error if the module doesn't have an `index.js` file and the `package.json`'s `main` field is not set.

That means that with the current logic it would go back to step 2, not 3, in the README.

However, I feel like it could be valuable to be able to publish package with multiple transforms. Do you expect users to to `-t someTransform/transforms/doA.js -t someTransform/transforms/doB.js` ?

If you want `-t someTransforms` to work, then some additional work needs to be done to get the package root. [find-root](https://www.npmjs.com/package/find-root) could help with that.